spec/models/manifest_spec.rb

```diff
@@ -97,4 +97,22 @@
       end
     end
   end
+
+  describe '#generator_mailing_address' do
+    context 'generator_mailing_address is present' do
+      it 'returns hash of generator_mailing_address' do
+        manifest = build(:manifest, content: { generator: { mailing_address: { zip_code: '12345' } } } )
+
+        expect(manifest.generator_mailing_address).to eq({ 'zip_code' => '12345' })
+      end
+    end
+
+    context 'generator_mailing_address is absent' do
+      it 'returns empty hash' do
+        manifest = build(:manifest)
+
+        expect(manifest.generator_mailing_address).to eq({})
```

Comment:
this brings up a larger question of whether we want to return keys for fields that have no data 

eg: return `{ 'zip_code' => '' }` here

I think it can be helpful to return empty values for all fields, but not something we have to tackle right now.